projects/storefrontlib/shared/components/media/media.config.ts

```diff
@@ -29,6 +33,54 @@ export abstract class MediaConfig {
     [format: string]: MediaFormatSize;
   };
 
+  /**
+   * Picture element configuration holds the media queries assigned to
+   * a format.
+   * The order of formats matters.
+   * <source> elements in <picture> will be sorted based on this order.
+   * This is necessary because the browser evaluates each
+   * <source> element in order and uses the first one that matches.
+   */
+  pictureElementFormats?: {
+    [format: string]: {
+      mediaQueries: PictureElementQueries;
+      width?: number;
+      height?: number;
+    };
+  };
+
+  /**
+   * Used to specify the order of formats.
+   * <source> elements in <picture> will be sorted based on this order.
+   * This is necessary because the browser evaluates each
+   * <source> element in order and uses the first one that matches.
+   *
+   * @example
+   * ['mobile', 'tablet', 'desktop']
+   */
+  pictureFormatsOrder?: string[];
+
+  /**
+   * A map that associates the keys of `PictureElementQueries` with their corresponding CSS media query features.
+   *
+   * @type {Record<keyof PictureElementQueries, string>}
+   *
+   * The `queryMap` is used to translate the query properties provided in a `PictureElementQueries` object
+   * to their respective CSS media query feature names. This map includes the following mappings:
+   *
+   * @example
+   * - `minWidth` -> `min-width`
+   * - `maxWidth` -> `max-width`
+   * - `minHeight` -> `min-height`
+   * - `maxHeight` -> `max-height`
+   * - `minDevicePixelRatio` -> `min-device-pixel-ratio`
+   * - `maxDevicePixelRatio` -> `max-device-pixel-ratio`
+   * - `orientation` -> `orientation`
+   * - `minAspectRatio` -> `min-aspect-ratio`
+   * - `maxAspectRatio` -> `max-aspect-ratio`
+   */
+  mediaQueryMap?: Record<keyof PictureElementQueries, string>;
```

Comment:
`major:` After secound thougt about `mediaQueryMap`, I'd consider removing it in favour of writing CSS media query features names directly in `pictureElementFormats`. Why:
- less logic in our `MediaService` because mapping is no logner required
- customers don't need to follow our opinionated syntax and can use CSS standards directly in the config

example
```ts
pictureElementFormats: {
    mobile: {
      mediaQueries: {
        'max-width': '767px',
      },
      width: 50,
      height: 50,
    },
    tablet: {
      mediaQueries: { 'min-width': '768px', 'max-width': '1024px' },
    },
}
```